### ISSUE TITLE
Tile: underlying HTML changes for a11y

### ISSUE THREAD

**Issue by jbadan** [SAP/fundamental-styles#111]
**Describe the bug**

See https://sap.github.io/fundamental/components/tile.html#tile-as-a-button. If it's a button, then it should be a button styled as a tile, not a `<div>` with a `role="button"`.

> Is this issue related to a specific component?

Tile

**Comment by droshev** [comment]
I think was related to the pre-Fiori tile component which doesn't exist anymore
